other/edge-server/client/README.md

```diff
@@ -34,32 +35,34 @@ previewing Edge Server, refer to the product page at
 - `install-deps`: installs the dependencies for the Express server and
   React client. You must install the Edge Server dependencies separately.
 - `start`: starts the Edge Server, Express server, and React client.
-- `shut-down`: shuts down the Edge Server, Express server, and React client.
+- `stop`: shuts down the Edge Server, Express server, and React client.
 
 ## Quick Start
 
-1. Create an App Services App based on the `edge-server.todo` template app, and
-   get Edge Server enabled for your new app.
+1. Create an App Services App based on the `edge-server.todo` template app.
 2. Download and configure the Edge Server.
-3. Start Docker
-4. From the project's root directory, run `npm run install-deps`.
-5. Add a `.env` file inside the `node-server` directory with the details
+3. As part of the Edge Server installation, choose to create a new user. Remember
+   the user's email and password for later.
+4. Start Docker
+5. From the project's root directory, run `npm run install-deps`.
+6. Add a `.env` file inside the `express-server` directory with the details
    required to run the Express Server.
 
    ```env
-   # This URI string is for Atlas Edge Server
-   EDGE_SERVER_URI="mongodb://localhost:27021"
+    # The Edge Server uses this connection string for anonymous authentication
+    EDGE_SERVER_URI="mongodb://localhost:27021"
 
-   # This port is for the Node.js server
-   PORT=5055
+    # This port is for the Express server
+    PORT=5055
    ```
 
-6. Then run `npm run start`.
-7. Explore how CRUD works from both the React Client and the App Services UI.
+7. Then run `npm run start`.
```

Comment:
I know this should go without saying, but more than once, I have cd'd into the `node-server` (now `express-server`) dir to add and populate the `.env` file, and forgotten to go back to the root directory to run `npm run start`.

```suggestion
7. From the project's root directory, run `npm run start`.
```